src/group_theory/perm/cycles.lean

```diff
@@ -426,6 +432,96 @@ begin
       (ih (λ τ hτ, h1 τ (list.mem_cons_of_mem σ hτ)) (list.pairwise_of_pairwise_cons h2)) },
 end
 
+section generation
```

Comment:
I think that some of these results deserve to be mentioned in module docstrings.
Could you please do that?

Reply:
Merci!